examples/plot_fpca.py

```diff
@@ -25,7 +25,18 @@
 # to explore datasets and obtain conclusions about said dataset using this
 # technique.
 #
-# First we are going to fetch the Berkeley Growth Study data. This dataset
+# FPCA is a dimensionality reduction method for functional data that aims to 
+# reduce the complexity of studying observations by finding a finite number of
+# principal components, which are the directions that capture the main modes 
+# of variation across the function (the most important directions in which the
+# curves vary). FPCA can be though of as a basis expansion, but what 
```

Comment:
Do not use such long sentences. In general it makes comprehension more difficult. Also "importance" is a very subjective word that does not capture what you try to say here:

```suggestion
# FPCA is a dimensionality reduction method for functional data that aims to 
# reduce the complexity of studying observations by finding a finite number of
# principal components. These components are the directions that capture the 
# main modes of variation across the function (the directions in which the
# curves vary the most). FPCA can be though of as a basis expansion, but what 
```